hadoop-tools/hadoop-distcp/src/test/java/org/apache/hadoop/tools/TestDistCpSync.java

```diff
@@ -1303,12 +1303,13 @@ public void testSyncSnapshotDiffWithDummyFileSystem() throws Exception {
       FileSystem dummyFs = FileSystem.get(URI.create("dummy:///"), conf);
       Assert.assertTrue(dummyFs instanceof DummyFs);
       new DistCp(conf, OptionsParser.parse(args)).execute();
+    } catch (UnsupportedOperationException e) {
+      Assert.fail("Dummy FS supports snapshots," +
```

Comment:
if you just rethrow e here, then the caller gets a full stack.

Reply:
Done.